ResourceStatusSystem/PolicySystem/Actions/LogPolicyResultAction.py

```diff
@@ -46,6 +46,7 @@ def run(self):
     statusType = self.decisionParams['statusType']
     if statusType is None:
       return S_ERROR('statusType should not be None')
+    #vo =self.decisionParams['vO']
```

Comment:
Is it here for debug?

Reply:
It must have been. The vo value is extracted a few lines below from the single policy result.